.ci/Jenkinsfile

```diff
@@ -206,8 +206,9 @@ pipeline {
                   withEnv(["HOME=${env.WORKSPACE}/${env.BASE_DIR ?: ''}"]) {
                     rubygemsLogin.withApi(secret: "${env.RELEASE_SECRET}") {
                       sshagent(['f6c7695a-671e-4f4f-a331-acdce44ff9ba']) {
-                        //sh "bundle install && rake release"
-                        sh 'ssh -T git@github.com | true'
+                        sh '.ci/prepare-git-context.sh'
+                        sh 'bundle install'
```

Comment:
I don't think we need the (rather heavy) bundle. Try without or perhaps with only `gem install rake` and then `rake release`

Reply:
- without the bundle 

I get something like the below error in my local environment
```
[Pipeline] sh
+ rake release
rake aborted!
LoadError: cannot load such file -- yard
```

- with `gem install rake`

```
+ gem install rake
Successfully installed rake-12.3.3
1 gem installed
[Pipeline] sh
+ rake release
rake aborted!
LoadError: cannot load such file -- yard
```

My local environment does use a kind of similar Gemfile:
- https://github.com/v1v/ruby-hello-world/blob/master/Gemfile

What should be do here?

Reply:
Looking inside `Rakefile`, `gem install rake yard rspec` should do it.

Reply:
Awesome, it worked like a charm

![image](https://user-images.githubusercontent.com/2871786/65268469-20a22380-db0f-11e9-879c-709481fee904.png)

Reply:
```suggestion
                        sh 'gem install rake yard rspec'
```